specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/preview/2023-10-01-preview/SAPVirtualInstance.json

```diff
@@ -0,0 +1,4455 @@
+          "SAPDatabaseInstances"
+        ],
+        "operationId": "SAPDatabaseInstances_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "SAPDatabaseInstances_Create": {
+            "$ref": "./examples/sapdatabaseinstances/SAPDatabaseInstances_Create.json"
+          },
+          "Create SAP Database Instances for HA System with Availability Set": {
+            "$ref": "./examples/sapdatabaseinstances/SAPDatabaseInstances_Create_HA_AvSet.json"
+          }
+        },
+        "description": "Creates the Database resource corresponding to the Virtual Instance for SAP solutions resource. <br><br>This will be used by service only. PUT by end user will return a Bad Request error.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SAPVirtualInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Request body of Database resource of a SAP system.",
+            "schema": {
+              "$ref": "#/definitions/SAPDatabaseInstance"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Describes the created Database resource of the SAP system.",
+            "schema": {
+              "$ref": "#/definitions/SAPDatabaseInstance"
+            }
+          },
+          "200": {
+            "description": "Describes the updated Database resource of the SAP system.",
+            "schema": {
+              "$ref": "#/definitions/SAPDatabaseInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SAPDatabaseInstances"
+        ],
+        "operationId": "SAPDatabaseInstances_Update",
+        "x-ms-examples": {
+          "SAPDatabaseInstances_Update": {
+            "$ref": "./examples/sapdatabaseinstances/SAPDatabaseInstances_Update.json"
+          }
+        },
+        "description": "Updates the Database resource.",
```

Comment:
Rephrase this as 
Updates the Database instance resource. This can be used to update tags on the resource.